heaps/min_heap.py

```diff
@@ -14,18 +16,29 @@ def __init__(self):
     def add(self, key, value = None):
         """ This method adds a HeapNode instance to the heap
             If value == None the new node's value should be set to key
-            Time Complexity: ?
-            Space Complexity: ?
+            Time Complexity: O(log n)
+            Space Complexity: O(1)
         """
```

Comment:
👍 However because `heap_up` is recursive, this method has an O(log n) space complexity.